server/repository/src/db_diesel/mod.rs

```diff
@@ -123,7 +123,7 @@ pub mod rnr_form_row;
 pub mod sensor;
 mod sensor_row;
 pub mod stock_line;
-mod stock_line_row;
+pub mod stock_line_row;
```

Comment:
Don't think you need this as there's a line 
`pub use stock_line_row::*;` right?
Can just refer to it as `crate::stock_line_row` or something like that?

That said i'm fine with this approach, I don't like the `pub use something::*` with IDE tools it's not needed and adds lot of lines of code...